build.js

```diff
@@ -0,0 +1,58 @@
+const _ = require('lodash');
+const chokidar = require('chokidar');
+const esbuild = require('esbuild');
+const glob = require('tiny-glob');
+const serveHandler = require('serve-handler');
+const http = require('http');
+
+const args = process.argv;
+const PORT = 3001;
+
+async function build(opts = {}) {
+  const start = Date.now();
+
+  /*
+  index.js is the entire library bundled up + exported in one file.
+  */
+  await esbuild.build({
+    entryPoints: ['./index.js'],
+    bundle: true,
+    outfile: './dist/index.js',
+    ...opts,
+  });
+
+  /*
+  For all the other files, we'll just export them to dist so folks can import the modules individually if they want.
+  */
+  let entryPoints = await glob('./src/js/**/*.js');
```

Comment:
Will this create a `dist/utils/` dir or just pull all js files up into the dist folder?

Reply:
This creates a `dist/utils` directory.

Reply:
Awesome. Is this ready to merge then?

Reply:
I think so! This should probably come with a decent version bump too, it _shouldn't_ change much for existing users, but it could. Once it's in and published to npm/unpkg I'll put together some examples.